functional_tests/scylla_operator/test_functional.py

```diff
@@ -860,3 +861,11 @@ def test_cloud_bundle_connectivity_cqlsh(db_cluster: ScyllaPodCluster):
 
     assert not res.stderr
     assert '(1 rows)' in res.stdout
+
+
+def test_can_recover_from_fatal_pod_termination(db_cluster):
+    target_node = db_cluster.nodes[-1]
+    experiment = PodFailureExperiment(pod=target_node, duration="30s")
```

Comment:
Why experiment? It sounds to me more from the your task point of view, but not test point of view.

In our case the `fault injection` is more suitable term here, IMHO.

Reply:
Experiment is the chaos-mesh terminology, I think it's o.k. to stick with that.

Reply:
We should not use each new terminology ` as-is` from each new lib we import.
There is well-known `adapter design pattern` for such things.

Reply:
I think `PodFailureExperiment` is a good name - first part shows that we are going to fail the pod, and the second `experiment` refers to chaos-mesh nomenclature (which is not bad). So it's clear we're going to fail the pod with chaos-mesh.

In case we will find a better tool for this purpose, we will anyway create different classes, that may need different approach, parameters, etc. and adapter might not fit here and create additional unnecessary work.

So my proposal is to introduce adapter when it is needed and continue with this approach.